src/Services/cdl/CdlLogLine.js

```
@@ -5,22 +5,11 @@ import {LINE_TYPE, LINE_TYPE_DELIMITER} from "./CDL_CONSTANTS";
-     * Given a log line, this function classifys and extracts its metadata.
-     * Note that the log line is in a format returned by the clp-ffi-js library.
-     * In this case, it is an array with the first element containing the
-     * log line. In the future, this can be optimized further.
-     */
-    _classifyLogLine () {
-        const fullStr = this.log[0].split("root ");
-        const log = fullStr.slice(1).join(" ").trim();
+    constructor (logLine) {
+        const pattern  = /^[.:a-zA-Z0-9_.-]+ [INFO|ERROR]+ adli (.*$)/gm;
+        const log = pattern.exec(logLine[0])[1]
 
```

ESLint

[error] Missing semicolon.

(semi)